users/users-router.js

```diff
@@ -3,6 +3,14 @@ const router = require("express").Router();
 const Users = require("./users-model.js");
 const restricted = require("../auth/restricted-middleware.js");
 
+router.get('/', restricted, (req, res) => {
+    Users.find()
+      .then(users => {
+        res.json(users);
+      })
+      .catch(err => res.send(err));
```

Comment:
Be sure to specify a status for errors; [more information on that is here](https://github.com/sinha-arpita/webauth-iii-challenge/pull/1#discussion_r271991476).